charts/bulk-scan-orchestrator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: bulk-scan-orchestrator
 apiVersion: v1
 home: https://github.com/hmcts/bulk-scan-orchestrator
-version: 0.2.9
+version: 0.2.10
```

Comment:
you will not need to bump the chart. as you will not add anything to values.yaml